libra_validation/fssh_based_2_state_models/recipes.py

```diff
@@ -157,3 +160,347 @@ def make_all_sets():
 
     return all_sets
 
+def set_recipe_v2(dyn_general, recipe, name=""):

```

Comment:
since, the recipe is mainly implied to be a synonym with "method", can we decouple initial conditions and model Hamiltonian from this? The recipes we define here should be transferable to other situation eventually - e.g. to atomistic simulations or file-based NBRA calculations